surveys/surveys.go

```diff
@@ -127,6 +143,39 @@ func getAvailableSchemasFromRegister() []LauncherSchema {
 	return schemaList
 }
 
+func GetAvailableSchemasFromCIR() []CIMetadata {
```

Comment:
We have some inconsistency in this file now, some is camel case some not, worth changing this and `GetLauncherSchema`. Or are there some different conventions used here?